tatt/packageFinder.py

```diff
@@ -1,17 +1,28 @@
 """module for extracting packages from a package/architecture list """
-
+import subprocess
 from .gentooPackage import gentooPackage as gP
 
-def findPackages (s, arch):
+def findPackages (s, arch, repo, bugnum=False):
```

Comment:
please also add the new params to the documentation

Reply:
Which documentation?

Reply:
Lines 6-8.

Reply:
I was a bit tired, I reckon. Done.